libcxx/cmake/caches/Generic-optimized.cmake

```diff
@@ -0,0 +1,4 @@
+set(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "")
+set(LIBCXX_TEST_PARAMS "optimization=speed" CACHE STRING "")
```

Comment:
Since it's possible to optimized for size too, I'd like "speed" in the name of the cache.